app/views/admin/deploy_groups/create_all_stages_preview.html.erb

```diff
@@ -0,0 +1,78 @@
+<h1>Creating Stages for <%= @deploy_group.name %> in <%= @deploy_group.environment.name %></h1>
+
+<h2>Creating Stages</h2>
```

Comment:
Looks a little different than the h1, and works to clarify the 2 tables seperately. See the screenshot.